app/models/mixins/relationship_mixin.rb

```diff
@@ -135,6 +135,13 @@ def parent_ids(*args)
     Relationship.resource_pairs(parent_rels(*args))
   end
 
+  # return a scope of all of the parents of this record (must :of_type with a single parameter)
```

Comment:
If of_type is required or single parameter, can you check and raise an ArgumentError (instead of the comment?)